charts/prometheus-snmp-exporter/values.yaml

```
@@ -215,6 +227,8 @@ serviceMonitor:
     #   relabelings: []
     # Map of metric labels and values to add. Overrides value set in serviceMonitor.additionalMetricsRelabels
```

Could you add this extra comment to highlight that this is for the existing 'replace' relabelling like you did on line 188.
```suggestion
    # Map of metric labels and values to add. Overrides value set in serviceMonitor.additionalMetricsRelabels
    # This sets fixed relabel configs with action 'replace'.
```